src/fromager/server.py

```diff
@@ -61,17 +60,22 @@ def serve_forever(server: http.server.ThreadingHTTPServer) -> None:
 
 
 def update_wheel_mirror(ctx: context.WorkContext) -> None:
-    logger.debug("updating wheel mirror")
     for wheel in ctx.wheels_build.glob("*.whl"):
-        logger.debug("adding %s", wheel)
-        shutil.move(wheel, ctx.wheels_downloads / wheel.name)
-    external_commands.run(
-        [
-            "pypi-mirror",
-            "create",
-            "-d",
-            os.fspath(ctx.wheels_downloads),
-            "-m",
-            os.fspath(ctx.wheel_server_dir),
-        ]
-    )
+        logger.info("adding %s to local wheel server", wheel.name)
+        downloads_dest_filename = ctx.wheels_downloads / wheel.name
+        # Always move the file so the code managing the timer for the
+        # wheels does not find more than one wheel in the build
+        # directory.
+        shutil.move(wheel, downloads_dest_filename)
+
+    for wheel in ctx.wheels_downloads.glob("*.whl"):
+        # Now also symlink the files into the simple hierarchy. We always
+        # process all files to be safe.
+        (normalized_name, _, _, _) = parse_wheel_filename(wheel.name)
+        simple_dest_filename = ctx.wheel_server_dir / normalized_name / wheel.name
```

Comment:
The normalized name wheel has underscores, the canonical PyPI name has dashes, e.g. `scikit-build-core/scikit_build_core-0.11.2-4-py3-none-any.whl`

Reply:
Please ignore my comment:

```python
>>> parse_wheel_filename("scikit_build_core-0.11.2-4-py3-none-any.whl")
('scikit-build-core', <Version('0.11.2')>, (4, ''), frozenset({<py3-none-any @ 140605344268096>}))
```